### PR TITLE
Add keyboard shortcuts and delete redirect

### PR DESCRIPTION
**Pull request by austinwbest**
#### Database Migration
NO

#### Description
New: Add support for left/right arrows on the movie details to navigate through movies
New: Add support for ctrl+home and ctrl+end to jump to the top and bottom of the movie index
New: Add redirect to previous movie instead of index when deleting one

### CODE REVIEW

File: frontend/src/Movie/Details/MovieDetails.js [hunk 1]
```diff
@@ -173,6 +174,15 @@ class MovieDetails extends Component {
     this.setState({ titleWidth: width });
   }
 
+  onKeyUp = (event) => {
+    if (event.keyCode === 37) {
```
Comment (Qstick) [inline frontend/src/Movie/Details/MovieDetails.js:178]: Add to keyCodes and use `import * as keyCodes from 'Utilities/Constants/keyCodes';`

**Comment by Qstick** [comment]
Should we add any of this to the keyboard shortcuts menu?

image

**Comment by austinwbest** [comment]
I don't see the need to. Ctrl + home /end are standard top to bottom keys & nothing specific to radarr. Arrow keys are standard nav keys so.. Just my opinion anyways.

Your call.

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ class DeleteMovieModalContent extends Component {
 
     this.state = {
       deleteFiles: false,
-      addImportExclusion: false
+      addImportExclusion: false,
+      previousMovie: ''
```
Comment (Qstick) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:28]: not needed in state I don't think, you are only using the value from props and it's never changed in this component

File: frontend/src/Movie/Delete/DeleteMovieModalContentConnector.js [hunk 3]
```diff
@@ -16,22 +17,26 @@ function createMapStateToProps() {
...
+  push
 };
 
 class DeleteMovieModalContentConnector extends Component {
 
   //
   // Listeners
-
-  onDeletePress = (deleteFiles, addImportExclusion) => {
+  onDeletePress = (deleteFiles, addImportExclusion, previousMovie) => {
```
Comment (Qstick) [inline frontend/src/Movie/Delete/DeleteMovieModalContentConnector.js:28]: You are essentially passing `previousMovie` from DeleteMovieModal, to this component then to `DeleteMovieModalContent` and back to here via this parameter... I think you can cut out ever handling it in `DeleteMovieModalContent` and just use the prop that is passed in from `DeleteMovieModal` here.... this avoids any change to `DeleteMovieModalConent` and avoids adding a parameter here.

**Review by Qstick — CHANGES_REQUESTED** [review]

File: frontend/src/Movie/Delete/DeleteMovieModalContentConnector.js [hunk 4]
```diff
@@ -16,14 +17,14 @@ function createMapStateToProps() {
...
-  deleteMovie
+  deleteMovie,
+  push
 };
 
 class DeleteMovieModalContentConnector extends Component {
 
   //
   // Listeners
-
```
Comment (Qstick) [inline frontend/src/Movie/Delete/DeleteMovieModalContentConnector.js:26]: Add this line back

**Pull request merged**